src/acquire.ts

```diff
@@ -36,10 +36,13 @@ export function acquire(
   item: Item,
   maxPrice?: number,
   throwOnFail = true,
-  maxAggregateCost?: number
+  maxAggregateCost?: number,
+  tryRetrievingUntradeable = false
```

Comment:
This might be getting to be a lot of additional optional parameters; it might be better to start adding an Options object instead